AppCenterCrashes/AppCenterCrashes/Internals/Util/MSCrashesCategory.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MSCrashesCategory : NSObject
```

Comment:
Please consider renaming the class.

Reply:
`MSApplicationForwarder` would align with other swizzling classes in the SDK.

Reply:
@clpolet not really. It's already aligned to analytics